cli/src/main/java/com/box/l10n/mojito/cli/command/L10nJCommander.java

```diff
@@ -206,7 +206,8 @@ public void createJCommanderForRun() {
         logger.debug("Create JCommander instance");
         jCommander = new JCommander();
 
-        jCommander.setAcceptUnknownOptions(true);
+        // TODO: Cant enable this until https://github.com/cbeust/jcommander/issues/377 is fixed
+        // jCommander.setAcceptUnknownOptions(true);
```

Comment:
I think that was used to pass extra configuration to spring ... not sure if it is actually used today

Reply:
@jeeyi Is this used on your side?